packages/office-ui-fabric-react/src/components/TeachingBubble/TeachingBubble.base.tsx

```diff
@@ -70,6 +70,8 @@ export class TeachingBubbleBase extends BaseComponent<ITeachingBubbleProps, ITea
       ? (classNames.subComponentStyles as ITeachingBubbleSubComponentStyles).callout
       : undefined;
 
+    const { onDismiss, ...rest } = this.props;
```

Comment:
cool trick :+1: I would do this though

```ts
const { onDismiss, ...rest } = this.props;
const { calloutProps: setCalloutProps, targetElement, isWide, styles, theme } = rest;
```

have them side by side in line 60, rather than apart over so many lines.